src/main/java/RacingCar/RacingCar.java

```diff
@@ -6,22 +6,37 @@
 public class RacingCar {
 
     private final List<Car> carList = new ArrayList<>();
-    public int tryCnt;
+    private int tryCnt;
 
-    public void initCars(int carCnt) {
-        for (int i = 0; i < carCnt; i++) {
-            carList.add(new Car());
+    RacingCar(String carName, int tryCnt) {
+        initCars(carName);
+        this.tryCnt = tryCnt;
+    }
+
+    private void initCars(String carName) {
+        String[] carNameList = carName.split(",");
+        for (int i = 0; i < carNameList.length; i++) {
+            carList.add(new Car(carNameList[i]));
         }
```

Comment:
```suggestion
        for (String car : carNameList) {
            carList.add(new Car(car));
        }
```
이렇게 하면 조금더 간결하고 보기 좋겠네요😃